examples/cosmosdb/account.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: cosmosdb.azure.jet.crossplane.io/v1alpha2
+kind: Account
+metadata:
+  name: example-account
+spec:
+  forProvider:
+    kind: MongoDB
+    location:  "East US"
+    offerType: Standard
+    mongoServerVersion: "4.0"
+    backup:
+    - type: Continuous
+    consistencyPolicy:
+    - consistencyLevel: Session
+    resourceGroupNameRef:
+      name: example
+    geoLocation:
+    - failoverPriority: 0
+      location: "East US"
+      zoneRedundant: False
```

Comment:
Btw, what happens if we omit this?

Reply:
For the `zoneRedundant` field, the default value is also `False` and when we omit this, the value is set to the `False`